hibernate-reactive-core/src/test/java/org/hibernate/reactive/BinInteger.java

```diff
@@ -1,23 +0,0 @@
-/* Hibernate, Relational Persistence for Idiomatic Java
- *
- * SPDX-License-Identifier: LGPL-2.1-or-later
- * Copyright: Red Hat Inc. and Hibernate Authors
- */
-package org.hibernate.reactive;
-
-import javax.persistence.AttributeConverter;
-import javax.persistence.Converter;
-import java.math.BigInteger;
-
-@Converter
-class BinInteger implements AttributeConverter<BigInteger,String> {
```

Comment:
What became of the test for JPA converters?

Reply:
THey are all in: [BasicTypesAndCallbacksForAllDBsTest](https://github.com/hibernate/hibernate-reactive/blob/abb3ecd984df9847a7b8c52ac897248f24c25039/hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/BasicTypesAndCallbacksForAllDBsTest.java)

Reply:
https://github.com/hibernate/hibernate-reactive/blob/abb3ecd984df9847a7b8c52ac897248f24c25039/hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/BasicTypesAndCallbacksForAllDBsTest.java#L256